examples/devices/lighty-actions-device/src/main/java/io/lighty/netconf/device/action/processors/ActionServiceDeviceProcessor.java

```diff
@@ -65,7 +67,7 @@ public QName getIdentifier() {
         final XmlElement fromDomElement = XmlElement.fromDomElement(requestXmlElement);
         final ActionDefinition actionDefinition = findActionInElement(fromDomElement);
 
-        if (actionDefinition.getQName().equals(Start.QNAME)) {
+        if (requireNonNull(actionDefinition).getQName().equals(Start.QNAME)) {
```

Comment:
Change method signature of `findActionInElement` to return Optional (instead of returing null)